src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
you should use https://github.com/component annotation only over implementations of interfaces, not over interfaces